### PR TITLE
Add timestamp to video frame in both input and output step

### PR DESCRIPTION
**Pull request by lubobill1990**
1. Add timestamp to VideoFrame type.
2. Send the in processing video frame's timestamp back

### CODE REVIEW

**Comment by MarcinJuraszek** [comment]
1. Has this been through the design review process?
2. You're changing parts of a capability that's public and not marked as beta. This is a breaking change.

**Review by MarcinJuraszek — CHANGES_REQUESTED** [review]
1. Has this been through a design review process?
2. You're changing APIs within a capability that's public and not marked as beta. This is a breaking change and as such can't be made in current form.

**Comment by lubobill1990** [comment]
@MarcinJuraszek I've rebased this PR to the latest main branch, and now it has the beta annotation.

**Review by TrevorJoelHarris — APPROVED** [review]

**Review by erinha — APPROVED** [review]

File: change/@microsoft-teams-js-a84b5246-c410-443c-a67b-87d06e89f7c9.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "add timestamp to VideoFrame type, send timestamp after video frame processed",
```
Comment (TrevorJoelHarris) [inline change/@microsoft-teams-js-a84b5246-c410-443c-a67b-87d06e89f7c9.json:3]: Our Contribution.md file says that change descriptions should be written in past tense and should surround code references in backticks.

Please update your comment to read:

"Added timestamp to `VideoFrame` type and return timestamp to the host after `VideoFrame` is processed"

Comment (lubobill1990) [reply change/@microsoft-teams-js-a84b5246-c410-443c-a67b-87d06e89f7c9.json]: Changed the description. Thanks!

**Review by TrevorJoelHarris — APPROVED** [review]

**Review by AE-MS — APPROVED** [review]
:shipit:

**Review by erinha — APPROVED** [review]

**Pull request merged**